local2pod.js

```diff
@@ -0,0 +1,46 @@
+import { SolidNodeClient } from "solid-node-client";
+const client = new SolidNodeClient;
+
+async function local2pod(){
+  try {
+    await client.login();
+  }
+  catch(e){ console.log(e); }
+  for(let file of files){
+    const from = `file:///`+process.cwd()+ '/' + file;
```

Comment:
does it depend on `cwd` where the script was called from?
i have a different approach in https://github.com/solid/catalog/blob/main/migrations/webid.js#L10-L12
if you want you could copy it and as part of #70 i will extract it into a helper